ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -130,6 +119,33 @@ bool WaylandWindow::Initialize() {
   return true;
 }
 
+void WaylandWindow::CreateXdgPopup() {
+  if (!parent_window_)
+    GetParentWindow();
```

Comment:
missing 'return'?

Reply:
No, return is not needed.

Reply:
it seems then that the GetParentWindow "populates" parent_window_, so name isn't ideal.

Reply:
ok!